package.json

```diff
@@ -128,6 +128,7 @@
     "jest-pact": "0.9.4",
     "lint-staged": "^13.0.0",
     "mini-css-extract-plugin": "^2.5.3",
+    "njk": "^5.0.0",
```

Comment:
@lvo700 why was this package added?

Reply:
I don't recall adding it. Unless it has something to do with reinstalling nunjucks after I uninstalled wsl

Reply:
Removed